docs/concepts/plugins.md

```diff
@@ -0,0 +1,93 @@
+---
+description: Plugins can add new functionality to Fresh without requiring significant complexity.
+---
+
+> Plugins are only available in Fresh 1.1 and above.
+
+Plugins can dynamically add new functionality to Fresh without exposing
+significant complexity to the user. Users can add plugins by importing and
+initializing them in their `main.ts` file:
+
+```ts
+// main.ts
+
+// ...
+
+import twindPlugin from "$fresh/plugins/twind.ts";
+import twindConfig from "./twind.config.js";
+
+await start(manifest, {
```

Comment:
Curious but what is manifest in this context? Might be a bit confusing for new readers.

Reply:
It is the `./fresh.gen.ts` file (see imports).

Reply:
I see the import was not there. I have added the imports for context now.